web/src/components/table/use-cases/scores.tsx

```diff
@@ -100,32 +106,85 @@ export default function ScoresTable({
         ) : null;
       },
     },
+    {
+      accessorKey: "traceName",
+      header: "Trace Name",
+      id: "traceName",
+      enableHiding: true,
+      enableSorting: true,
+      cell: ({ row }) => {
+        const value: string = row.getValue("traceName");
+        const filter = encodeURIComponent(
+          `name;stringOptions;;any of;${value}`,
+        );
+        return (
+          <TableLink
+            path={`/project/${projectId}/traces?filter=${value ? filter : ""}`}
+            value={value}
+            truncateAt={40}
+          />
+        );
+      },
+    },
     {
       accessorKey: "timestamp",
       header: "Timestamp",
+      id: "timestamp",
       enableHiding: true,
+      enableSorting: true,
     },
     {
       accessorKey: "name",
       header: "Name",
+      id: "name",
       enableHiding: true,
+      enableSorting: true,
     },
     {
       accessorKey: "value",
       header: "Value",
+      id: "value",
       enableHiding: true,
+      enableSorting: true,
+      cell: ({ row }) => {
+        const value: number = row.getValue("value");
+        return value % 1 === 0 ? value : value.toFixed(4);
+      },
+    },
+    {
+      accessorKey: "userId",
+      header: "User ID",
+      id: "userId",
+      enableHiding: true,
+      enableSorting: true,
+      cell: ({ row }) => {
+        const value = row.getValue("userId");
+        return typeof value === "string" ? (
+          <>
+            <TableLink
+              path={`/project/${projectId}/users/${value}`}
+              value={value}
+              truncateAt={40}
+            />
+          </>
+        ) : undefined;
+      },
     },
     {
       accessorKey: "comment",
       header: "Comment",
+      id: "comment",
       enableHiding: true,
+      enableSorting: true,
```

Comment:
Can you remove sorting here? I think sorting by free text is quite expensive for the db